crates/rolldown_debug/src/init_tracing.rs

```diff
@@ -8,6 +9,8 @@ use tracing_subscriber::prelude::*;
 use crate::build_id_propagate_layer::BuildIdPropagateLayer;
 use crate::debug_formatter::DevtoolFormatter;
 
+static TRACER_ID: AtomicU32 = AtomicU32::new(0);
```

Comment:
What is this used for? Did this mean there may exist multiple `DebugTracer` at the same time?